MODULE.bazel

```diff
@@ -0,0 +1,8 @@
+module(
+    name = "yams",
```

Comment:
unfortunately bzlmod is prescriptive about this and you cannot have capital letters

Reply:
it's fine